### PR TITLE
Added "Start" button

### PR DESCRIPTION
**Pull request by Gr3g027**

### CODE REVIEW

File: mainwindow.ui [hunk 1]
```diff
@@ -14,26 +14,59 @@
...
+   </widget>
   </widget>
   <widget class="QMenuBar" name="menubar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
-     <height>25</height>
+     <height>22</height>
```
Comment (Sclafus) [inline mainwindow.ui:69]: why tho 🤨

File: mainwindow.h [hunk 2]
```diff
@@ -43,6 +45,7 @@ class MainWindow : public QMainWindow
 
 	private:
 		QListWidgetItem *getTwitchChatMessage(QString username, QString message);
+		QPushButton *btnStart;
```
Comment (Sclafus) [inline mainwindow.h:48]: btnStart is not really a good name imho

File: mainwindow.cpp [hunk 3]
```diff
@@ -94,12 +95,20 @@ void MainWindow::on_actionAbout_triggered()
 {
 	// TODO: DEBUG: move this
 	qDebug() << "about";
+	// twitchClient->init();
+	// gosumemoryClient->init();
+	// osuIrcClient->init();
+}
+
+void MainWindow::on_actionInit_triggered()
```
Comment (Sclafus) [inline mainwindow.cpp]: it's not a QAction, why is it called `on_action[...]`?

**Review by Sclafus — APPROVED** [review]
Thanks, I'm  just nitpicking

File: mainwindow.cpp (hunk 3 above)
Comment (Gr3g027) [reply mainwindow.cpp]: You are right, I was just following the "on_action" signals but I'll change it

File: mainwindow.h (hunk 2 above)
Comment (Gr3g027) [reply mainwindow.h:48]: I'm not good at giving names. How should it be called? 🙃

File: mainwindow.ui (hunk 1 above)
Comment (Gr3g027) [reply mainwindow.ui:69]: I actually don't know hahaha I assumed that was the changes made to the whole ui file, I just messed up my commits

**Pull request merged**